### PR TITLE
[SOLUTION] calculate operations sum in corrupted memory

### PR DESCRIPTION
**Pull request by SaidNajim**

### CODE REVIEW

**Review by ilyasabdellaoui — APPROVED** [review]

File: bash/day03/script.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+input="input.txt"
```
Comment (jamali005) [inline bash/day03/script.sh]: I think the name of the file is corrupted_memory.txt not input.txt ?

**Review by jamali005 — COMMENTED** [review]
change the name of input file i guess

File: bash/day03/script.sh (hunk 1 above)
Comment (SaidNajim) [reply bash/day03/script.sh]: Yes, I renamed the text file but forgot to update the code. Thanks for noticing

**Comment by jamali005** [comment]
Thanks @SaidNajim 🙏

**Comment by SaidNajim** [comment]
@jamali005 Please approve if everything is all right

**Review by jamali005 — APPROVED** [review]

**Pull request merged**